circle.yml

```diff
@@ -0,0 +1,55 @@
+machine:
+  environment:
+    ARCH: "i686"
+    BUILDOPTS: "-j3 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
+    TESTSTORUN: "all"
+    JULIA_CPU_CORES: 2
```

Comment:
I did some science and if my analysis is correct, you get a total of 4 containers at any given time on a free account. So with 4x parallelism, there are no containers left for other jobs. With 2x, there are 2 available containers left, so you get 1 other job with 2x parallelism. With 1x, there are 4 containers available for 4 jobs. 1x seems to take too long for it to be a viable replacement for Travis.